pandas/tseries/resample.py

```diff
@@ -499,12 +499,12 @@ def interpolate(self, method='linear', axis=0, limit=None, inplace=False,
                                   limit_direction=limit_direction,
                                   downcast=downcast, **kwargs)
 
-    def asfreq(self):
+    def asfreq(self, **kwargs):
```

Comment:
Could you say why your doing `**kwargs` here instead of just `fill_value=None`? If that's consistent with other methods, then no problem.